docs/getting-started.md

```diff
@@ -84,6 +69,45 @@ public class MainActivity extends ReactActivity {
 }
 ```
 
+If you're on React Native >= 0.60, you also need to do the following:
+
+[Disable autolinking for `react-native-gesture-handler`](https://github.com/react-native-community/cli/blob/master/docs/autolinking.md#how-can-i-disable-autolinking-for-unsupported-library). To disable autolinking for it, create a `react-native.config.js` file in the root of your project with the following content:
+
+```js
+module.exports = {
+  dependencies: {
+    'react-native-gesture-handler': {
+      platforms: {
+        android: null,
+        ios: null,
+      },
+    },
+  },
+};
+```
+
+Use [jetifier](https://github.com/mikehardy/jetifier) to work around libraries using the support library on Android:
```

Comment:
How about rephrasing:

You can include it in your "postinstall" step of "package.json" file:
```json
{
  "postinstall": "npx jetifier"
}
```
This way you can avoid installation steps